### ISSUE TITLE
Model Computation Time

### ISSUE THREAD

**Issue by AlgorithmicIntelligence** [kaixin96/PANet#14]
I evaluate the processing time on inference with the line of code as below.
'''
time_s = time.time()
query_pred, _ = model(support_images, support_fg_mask, support_bg_mask,
                                      query_images)
time_elapsed = time.time() - time_s
'''
I just found that the FPS of the model can above 30, is that really?
Because I though instance segmentation usually time-consuming.

**Comment by kaixin96** [comment]
Hi @AlgorithmicIntelligence , I just checked previous testing log. For 1-way 1-shot setting, the wall clock time for evaluating on 1000 episodes (on a single Titan X) is 5 mins and for 1-way 5-shot it is 8 mins. Hope this information helps.

I did not measure the FPS of our model but 30 is too fast to be true. By the way, our work is in semantic segmentation category instead of instance segmentation.

Thank you :)
